app/src/main/java/com/example/triptracker/view/profile/UserProfileOverview.kt

```diff
@@ -67,199 +72,264 @@ fun UserProfilePreview() {
  */
 @Composable
 fun UserProfileOverview(
-    // userProfileViewModel: UserProfileViewModel = UserProfileViewModel(),
+    userProfileViewModel: UserProfileViewModel = UserProfileViewModel(),

```

Comment:
I think that the line should be : userProfileViewModel: UserProfileViewModel = viewModel() so that the view model is not reloaded each time